src/Package/Impl/DataInspect/Office/CsvAppFileIO.cs

```diff
@@ -71,6 +73,22 @@ internal static class CsvAppFileIO {
             Interlocked.Exchange(ref _busy, 0);
         }
 
+        public static void Close() {
+            var folder = GetTempCsvFilesFolder();
+            if (Directory.Exists(folder)) {
+                foreach (var f in Directory.EnumerateFiles(folder)) {
+                    try {
+                        File.Delete(f);
+                    } catch(IOException) { } catch (UnauthorizedAccessException) { }
+                }
+            }
+        }
+
+        private static string GetTempCsvFilesFolder() {
+            var folder = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);
```

Comment:
we dont know all the different scenarios that users may have for RTVS-Excel.  for example, someone might want to use this feature to export large DF's or a series or related DF's to Excel to do some Excel centric processing with them post RTVS session.  as such, you dont want them to go hunting around in their TEMP folder digging up .csv files.  i personally think the 1) location and 2) whether they're deleted or not should be options with defaults: 1) ~/RTVS-Excel/ and 2) Delete on Exit.